roles/provision-prometheus-apb/tasks/main.yml

```diff
@@ -101,6 +101,31 @@
           name: '{{ prometheus_configmap_volume_name }}'
         - mount_path: /prometheus
           name: prometheus-volume-1
+    volumes:
+      - name: '{{ prometheus_configmap_volume_name }}'
+        config_map:
+          defaultMode: 420
+          name: '{{ prometheus_configmap_name }}'
+      - name: prometheus-volume-1
+        persistent_volume_claim:
+          claim_name: prometheus-claim
+
+- name: create grafana deployment config
```

Comment:
Would it be interesting to split these into a separate `provision-grafana-apb` role?

Reply:
@paolobueno yes this is a great suggestion!